lib/schema_plus/foreign_keys/helpers/compatibility.rb

```diff
@@ -0,0 +1,26 @@
+module SchemaPlus::Compatibility
```

Comment:
Very nice idea to make a separate module to handle the compatibility things. Such a nice idea that I think it ought to be part of schema_plus_core -- or better yet its own separate gem.   It will be handy in other gems, I know it would have cleaned up the testing in `schema_plus_indexes` (and if I remember right it would only be a development dependency, which is nice)